rts/Sim/Units/Unit.cpp

```diff
@@ -1339,7 +1339,7 @@ void CUnit::ApplyImpulse(const float3& impulse) {
 	if (!moveType->CanApplyImpulse(modImpulse))
 		return;
 
-	CSolidObject::ApplyImpulse(modImpulse);
+	CSolidObject::ApplyImpulse(modImpulse, moveType->GetMaxSpeed() * 1.2);
```

Comment:
why 1.2? wouldn't having basically no limit be fine if the unit brakes to its proper speed anyway?

Reply:
Why is this multiplier here at all? What is it addressing?